src/main/java/Parser.java

```diff
@@ -0,0 +1,98 @@
+/** Parser class Responsible for parsing user commands */
+public class Parser {
+
+  /**
+   * Takes in user commands in a tokenized string form Updates tasklist accordingly
+   *
+   * @param inputs User inputs
+   * @param tasks TaskList
+   */
+  public void parseCommand(String[] inputs, TaskList tasks) {
+    String command = inputs[0];
+    switch (command) {
```

Comment:
Should there be no need for indentation in switch statements?

Reply:
Same reason as the one above
https://google.github.io/styleguide/javaguide.html#s4.8.4-switch